custom_components/spotcast/spotify_controller.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, access_token=None, expires=None):
         self.device = None
         self.credential_error = False
         self.waiting = threading.Event()
+        self.castDevice = castDevice
```

Comment:
purely for future PR, I'm going to change it directly, I maintain the pep8 format with snake_case for variables and function name